### PR TITLE
User Feedback Session:Enterprise Focus Group

### PR DESCRIPTION
**Pull request by mihaiep**
minutes/notes

### CODE REVIEW

File: meetings/2019-04-25.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+* Call for Actions:
+    * Package Maintenance
+    * Node Releases: better understand Enterprise plans of adoption and barriers
+    * v6 goes EOL in 1 week
+    * v12 released on April 23
+    
+## Invited
+* User Feedback Members: @nodejs/user-feedback
+* Enterprise Group Members:
+  * Aaron Heckmann <@aheckmann> - @paypal
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: extra space here?

File: meetings/2019-04-25.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+### Notes:
+* Thank you for participating in this meeting!
+* Intro
+* Ahmad: How big are the teams in your company and roughly how many packages/software distributions? In in Telus we had 500people
+  with  two third software engineers and 1700 software applications about 900 using Node.JS
+* Dan: At PayPal we are 22000 and 5000 engineers we are about half Node half Java.
+* Brian: we are about 131000 employees and Node it’s one of our biggest platforms.
+* Dominycas: the projects I’m envolved right now is fairly small for enterprise but we have very struct security and audit requirements.
+* Keith: I’m evangelist in HP for the Node.Js  at the enterprise level and working with a lot of clients using node, so my
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: Node.js as opposed to Node.Js and then extra space.

File: meetings/2019-04-25.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  for the teams, I’m interesting to hearing if your companies have any interest or have surfaced any patterns in terms of keeping 
+  up to date with Node.Js security patches and releases and EOL for older versions of node. 
+* Luke: In the terms of keeping up to day, keeping on top of the Twitter or distributions groups , so there isn’t a sort of single
+  unified way of getting all of that information.
+* Ahmad: Is there anything Node project can do different in terms of not just announcing something and posting on Twitter but in 
+  giving you that roadmap and maybe a different file structure or different visibility that help you take that back to your teams
+  and roll that part of your planning of your work.
+* Michael: The one thing I do want to that though is that beyond being on the repo which has the specific date, it has been every April,
+  every October for like over three years, it’s very interesting people still don’t know that because that’s like part we’re trying 
+  to achieve. You can just plan for April.
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: you can just plan for April -> You can just plan for April and October.

File: meetings/2019-04-25.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  the cycle what we do is build a little contingency into our planning and just accept the fact there’s going to be updates not just 
+  from node but from a whole range of other tools that we use. 
+* Keith: Most of my customers are financial services so I think it’s a general trend the weight analyst is basically a very strict
+  cadence and the bigger issue for more clients that I work with, is the cadence can’t be spread out too far so if the releases fall
+  two or three behind, that becomes an issue. Security patches trump all the other stuff.
+* Michael: There is always going to be ad-hoc high security releases for things like open SSL has vulnerabilities, we are discussing
+  internally whether something like a regular schedule for lower vulnerability security releases would make sense, so I’d be interested
+  in this group input on that in terms of if we plan do those every say four months and roll things up in to them, so basically some 
+  things may come out later, some things may not come out at all until the next thing, but that’s probably better than what we do today
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: until the next release

File: meetings/2019-04-25.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+  which is basically once we get a critical mass that we think is enough to make a security release even there aren’t high priority
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: even though they aren't

File: meetings/2019-04-25.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+  versions of node and that is largely dictated to us by some of the cloud vendors that we use in supported versions of the node. 
+* Ahmad: There are a lot of Node.Js 6 dependencies or deployments out there in production how are you planning on addressing
+  that in 2 weeks (EOL of version6)?
+* Steve: We identified about 25 packages still running Node 6 and like 20 of them can just get banned.
+* Ahmad: Have any of your organization or your development teams been able to articulate in one shape or form what is a total cost of 
+  ownership for Node.Js based application looks like or a formula around that? Given the complexity of dependency management,
+  given the complexities of the web and backend and giving the complexities of knowledge management around those teams,
+  there are lessons for Node project to take away, as part of documentation, as part of training materials as part of management 
+  solutions that the Node foundation can invest more and just create if they don’t exist, to help organizations.
+* Michael: I’d be interested like the specific thing that was mentioned that even turning the flag on requires rolling things out,
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: I'd be interested in the specific thing that was mentioned which was that

File: meetings/2019-04-25.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+  so any feedback on small changes that might help make that easier so for example if you actually take the binaries vet them
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: easer.  For example

File: meetings/2019-04-25.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+  “well these are the features are build in node and these are the libraries that this particular enterprise organization uses
+  across the board so therefore you can get them from a private npm repo and they’re supported and they’re security checked.
+  Its kind like of a hidden cost, but it is also great strength because you don’t have contend with something that’s 
+  in the standard library you don’t want to have, you just bring what you need.
+* Ahmad: What is the business impact of that?
+* Luke: Different organizations using libraries in npm the question is around whether we kind of provide some kind of badging 
+  to that when a significant proportion of enterprise consumers of node are using some libraries and we can sort of service that
+  in a more visible way generally projects that are quite proud of who uses them they talk about themselves all the time in their
+  github repos, but having that kind of data more actively surfaced in a kind of rating system would probably help.
+* Michael: I think that fit in the package maintenance team where the effort is to figure it out how we bring people together to make
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: that fits

File: meetings/2019-04-25.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+* Luke: I think it’s about surfacing risk. What we can do is to ensure the information is available in a consistent form.
+* Ahmad: As representatives do you find any challenge in making the case for those type of investments and what are the 
+  challenges you’re facing?
+* Steve: everything at Telus relates around the business case for what it is better than what is currently there, c
+  onversation about what at the enterprise level, new versions like new version node will enable to do offer for us, 
+  I think is a really useful thing that Foundation can have like a media package.
+* Ahmad: performance is a good example, so the teams can work on at every release of Node.Js.
+* Michael: we already have a benchmark group who generates some charts, the challenges, all those things are very valuable,
+  we would just need more people to engage to help them happen, because based on the current people who are involved we’re not 
+  going to be able to do that.  The sort of framework group is there we just need some involvement if that’s one of priorities.
```
Comment (mhdawson) [inline meetings/2019-04-25.md]: The group is there, we just need more involvement if...

**Comment by mihaiep** [comment]
Thank you Michael, for corrections/inputs.

**Pull request merged**